DNN Platform/DotNetNuke.Web/Api/Internal/DnnActionFilterProvider.cs

```diff
@@ -3,17 +3,25 @@
 // See the LICENSE file in the project root for more information
 
 namespace DotNetNuke.Web.Api.Internal
-{
+{
+    using System;
     using System.Collections.Generic;
     using System.Linq;
     using System.Web.Http;
     using System.Web.Http.Controllers;
     using System.Web.Http.Filters;
 
-    using DotNetNuke.Common;
+    using DotNetNuke.Common;
+    using DotNetNuke.DependencyInjection.Extensions;
 
     internal class DnnActionFilterProvider : IFilterProvider
-    {
+    {
+        private IServiceProvider container;
+        public DnnActionFilterProvider(IServiceProvider container)
+        {
+            this.container = container;
+        }
+
         public IEnumerable<FilterInfo> GetFilters(HttpConfiguration configuration, HttpActionDescriptor actionDescriptor)
         {
             // Requires.NotNull("configuration", configuration);
```

Comment:
While we are in here can we remove the commented out code

Reply:
Yes we can, no problem

Reply:
Removed stale code in latest update